### DISCUSSION TITLE
presentation for meetup

### DISCUSSION THREAD

**Discussion by btsuhako** [apache/apisix#9789] [General]
we have a local Kubernetes meetup group that's interested in hearing about APISIX. who is best to reach out to about this? thanks!

**Comment by juzhiyuan** [comment]
Hello @btsuhako, feel free to contact @juzhiyuan, @Yilialinn, @Boburmirzo, or @moonming :)

My email: juzhiyuan@apache.org

My availability: https://meetings.hubspot.com/zhiyuan

**Reply by pottekkat** [reply]
@juzhiyuan @btsuhako We have @nfrankel traveling to the US in the coming days. Maybe we can arrange something?

**Reply by juzhiyuan** [reply]
That's cool! Yes

**Reply by nfrankel** [reply]
I'll be in San Matteo up to Aug. 18th and I need to be in Seattle on the 22nd.

I could potentially be in Salt Lake City between those 2 dates but I need to have a firm commitment before mid-next week as I'll book plane tickets and hotels.

**Reply by btsuhako** [reply]
thanks everyone!

@nfrankel I'll send you an email to coordinate some details. you're welcome to come visit us in person in Salt Lake, and we do have a setup for remote presenters
